### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by cg-cnu**
Hey @krathjen 
Just made some changes to readme. 
Organized the data a little and added some from the website. 
Feel free to suggest or make more changes. 
Hope it helps! Thanks 😄

### CODE REVIEW

**Comment by krathjen** [comment]
Thanks @cg-cnu! I appreciate the help and the changes are looking good!

I'll suggest some other changes soon.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
-## Studio Library
+<p align="center">
```
Comment (krathjen) [inline README.md]: Would it be possible to have the icon to the left of the title and then align the title to the left?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+  <a href="http://www.studiolibrary.com/">
+    <img
+      alt="logo"
+      src="./resource/icons/icon_white_on_blue.png"
+    />
+  </a>
+  <h1 align="center">Studio Library</h1>
+  <h4 align="center">Animation and pose manager for Maya</h4>
```
Comment (krathjen) [inline README.md]: Let's remove "Animation and pose manager for Maya" and just have the description.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
+</p>
 
-Studio Library is a free python script for managing poses and animation in Maya.
 
-* www.studiolibrary.com
+## Description :
```
Comment (krathjen) [inline README.md]: I don't think we need the "Description" header, so maybe just have "A Free tool ..." under the title without the header.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
 
-* https://github.com/krathjen/studiolibrary
+A Free tool written in python for managing poses and animation in Maya.
 
 
-### Features
+## Features :
```
Comment (krathjen) [inline README.md]: Let's remove the ":" from all the subheaders. I think this looks a bit cleaner.

**Comment by cg-cnu** [comment]
Hey @krathjen made the changes mentioned in the comments.
* Did a left alignment for log and text 
* Removed the tag line
* removed description header 
* removed `:` for more clarity

**Comment by krathjen** [comment]
Thanks @cg-cnu! The update looks good!! I'll merge this with the develop branch for now.

**Pull request merged**